profiling/src/internal/profile.rs

```diff
@@ -12,6 +12,7 @@ use std::collections::HashMap;
 use std::time::{Duration, SystemTime};
 
 pub struct Profile {
+    backup_sample_types: BackupSampleTypesAndPeriod,
```

Comment:
Minor: This seems "weird enough" to deserve a bit of a hint ;)

```suggestion
    // used when resetting the profile, see struct definition for details
    backup_sample_types: BackupSampleTypesAndPeriod,
```

Reply:
Removed :D